qfieldcloud_sdk/sdk.py

```diff
@@ -30,26 +30,64 @@
 
 
 DEFAULT_PAGINATION_LIMIT = 20
+"""int: Defines the default limit for pagination, set to `20`."""
 
 
 class FileTransferStatus(str, Enum):
+    """Represents the status of a file transfer.
+
+    Attributes:
+        PENDING (str): The transfer is pending.
+        SUCCESS (str): The transfer was successful.
+        FAILED (str): The transfer failed.
+    """
+
     PENDING = "PENDING"
     SUCCESS = "SUCCESS"
     FAILED = "FAILED"
 
 
 class FileTransferType(Enum):
```

Comment:
Should this also not be a str, Enum?
```suggestion
class FileTransferType(str, Enum):
```

Reply:
Good spot! Ideally should be https://docs.python.org/3/library/enum.html#enum.StrEnum, but it's 3.11 onwards (I am looking for bigger backwards compatibility for this kind of libs).